src/features/shitcoin.ts

```diff
@@ -133,7 +138,8 @@ export class ShitcoinFeature extends GlobalFeature {
         }
         const oldPrice = await this.bot.brain.get(BRAIN_KEYS.CURRENT_PRICE)
         const btcPriceData = await this.getPriceFromCoinbase(CURRENCY_CODE)
-        await this.bot.brain.set(BRAIN_KEYS.CURRENT_PRICE, btcPriceData.data.amount)
+        const aussieDollarBucks = NUMBER_FORMATTER.format(Number(btcPriceData.data.amount))
+        await this.bot.brain.set(BRAIN_KEYS.CURRENT_PRICE, aussieDollarBucks)
```

Comment:
Best not to _persist_ the formatted amount. The raw numbers should be persisted and this formatting should happen at the time the UI strings are generated (up in the `messageEmbed()` function)